tuning/sft_trainer.py

```diff
@@ -262,6 +262,9 @@ def train(
         if tokenizer.unk_token is None:
             logger.warning("UNK token set to default, missing in tokenizer")
             special_tokens_dict["unk_token"] = configs.DEFAULT_UNK_TOKEN
+        if tokenizer.pad_token == tokenizer.eos_token:
+            logger.warning("PAD token set to default, different from eos token")
+            special_tokens_dict["pad_token"] = configs.DEFAULT_PAD_TOKEN
```

Comment:
@Ssukriti @Abhishek-TAMU what if the eos_token in tokenizer is actually the DEFAULT_PAD_TOKEN? Can we handle that case?

Reply:
example - 
tokenizer.eos_token is configs.DEFAULT_PAD_TOKEN
and 
tokenizer.pad_token is configs.DEFAULT_PAD_TOKEN

At this point, we are setting tokenizer.pad_token  to configs.DEFAULT_PAD_TOKEN this again. We might want to think about this case.

Reply:
Good point. In this case can we modify and assign `eos_token` as `configs.DEFAULT_EOS_TOKEN` ? @Ssukriti

Reply:
its very unlikely that EOS token will be <PAD> no?

Reply:
but ya if we want to be absolutely sure, we can do that

if tokenizer.pad_token == tokenizer.eos_token:
     if tokenizer.eos_token != configs.DEFAULT_PAD_TOKEN:
            tokenizer.pad_token  = configs.DEFAULT_PAD_TOKEN
     else:
           tokenizer.eos_token = configs.DEFAULT_EOS_TOKEN